doc/classes/Object.xml

```diff
@@ -905,6 +905,13 @@
 				[b]Note:[/b] Metadata that has a name starting with an underscore ([code]_[/code]) is considered editor-only. Editor-only metadata is not displayed in the Inspector and should not be edited, although it can still be found by this method.
 			</description>
 		</method>
+		<method name="remove_user_signal" experimental="">
+			<return type="void" />
+			<param index="0" name="signal" type="StringName" />
+			<description>
+				Removes the given user signal [param signal] from the object. See also [method add_user_signal] and [method has_user_signal].
```

Comment:
Type punning like I suggest is actually not recommended (not great for translations), but for consistency with the other methods I'd use it anyway.
```suggestion
				Removes the given user [param signal] from the object. See also [method add_user_signal] and [method has_user_signal].
```